protelis/protelis-lang/src/main/protelis/protelis/coord/accumulation.pt

```diff
@@ -59,7 +59,7 @@ public def boundAverage(sink, local, range, null) {
  * @return          T, aggregated value
  */
 public def C(potential, reduce, local, null) {
-    rep (v <- local) {
+    share (v,nbr_v <- local) {
```

Comment:
Do we need  `v`? Can we drop it and only use `nbr_v`? (even better, remove `nbr_v` and rename its usages to `v`)

Reply:
Done.  That was a left-over artifact from when I was figuring out the that the subfunction `v` wasn't the same `v`